docusaurus/docs/how-to-guides/app-plugins/fetch-data-from-frontend-to-app.md

```diff
@@ -0,0 +1,132 @@
+---
+id: fetch-data-from-frontend-to-app
+title: Fetch data from frontend code to app plugin using the data proxy
+description: Learn how to use the data proxy API to fetch data from frontend code in app plugins in Grafana
+keywords:
+  - grafana
+  - plugins
+  - data proxy
+  - frontend
+  - app
+  - app plugin
+  - CORS
+---
+
+# Fetch data from app plugins
+
+Along with the JavaScript [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), the Grafana data proxy is used to fetch data from a Grafana app plugin.
+
+The data proxy is especially useful
+
+- for overcoming cross-site (CORS) limitations, or
+- for performing authenticated requests, or
+- for sending other sensitive data from your plugin configuration to Grafana.
+
+This guide explains how the data proxy works in app plugins and explores common issues in its usage. For usage related to data source plugins, refer to our [documentation](../data-source-plugins/fetch-data-from-frontend.md)
+
+## How does it work?
```

Comment:
I noted that most or all of the content here until the specific app example is a copy of the other article. could you instead create a shared file and use it here? that way we don't have to maintain two different articles with the same content.